### PR TITLE
update zenodo

### PR DESCRIPTION
**Pull request by bobmyhill**
Updates the zenodo link to point to the new repository.

### CODE REVIEW

File: homepage/download.html [hunk 1]
```diff
@@ -54,11 +54,13 @@ <h2>A thermodynamic and geophysics toolkit for the Earth and planetary sciences<
...
 <h3>Download:</h3>
 
+The links below are for the latest released version of BurnMan. You may also wish to check out the latest development version, which you can access by clicking the "Source" link above.
 <p>
 <ul>
-<li><a href="https://github.com/geodynamics/burnman/releases/tag/v0.9.0">Latest release:</a> <a href="https://github.com/geodynamics/burnman/releases/download/v0.9.0/burnman-0.9.0.zip">burnman-0.9.0.zip</a> </li>
-<li><a href="doc-0.9.0/">Online Documentation</a></li>
-<li><a href="https://github.com/geodynamics/burnman/releases/download/v0.9.0/BurnMan-0.9.0.pdf">PDF Manual</a></li>
+<li><a href="https://github.com/geodynamics/burnman/releases/tag/v0.10.0">Latest release:</a>
+    <a href="https://github.com/geodynamics/burnman/releases/download/v0.10.0/burnman-0.10.0.zip">burnman-0.10.0.zip</a> </li>
```
Comment (tjhei) [inline homepage/download.html:61]: This files does not exist, yet, only https://github.com/geodynamics/burnman/archive/refs/tags/v0.10.0.zip gets generated automatically. I don't like the filenames, so I would download, rename. and reupload the files to the release.

File: homepage/index.html [hunk 2]
```diff
@@ -97,18 +97,19 @@ <h3>Features</h3>
 <h3>Citations</h3>
```
Comment (tjhei) [inline homepage/index.html:97]: I currently need to update website manually. Would you mind if we move it to github pages? We are no longer doing anything special/interactive with the website, anyways.

File: homepage/download.html (hunk 1 above)
Comment (bobmyhill) [reply homepage/download.html:61]: Thanks! Done.

File: homepage/index.html (hunk 2 above)
Comment (tjhei) [reply homepage/index.html:97]: https://geodynamics.github.io/burnman/

Comment (bobmyhill) [reply homepage/index.html:97]: Cool, thank you! Ok, I'll work on updates to the links in the docs / on zenodo etc etc.

Comment (tjhei) [reply homepage/index.html:97]: you will need to push to the newly created www branch (or make PRs if you want things to be reviewed)

Comment (bobmyhill) [reply homepage/index.html:97]: Ok, great. In that case, shall we remove the homepage directory from the repo?

Comment (tjhei) [reply homepage/index.html:97]: Yes, that makes sense to me.

**Review by tjhei — APPROVED** [review]

**Comment by tjhei** [comment]
Website looks great!

**Comment by bobmyhill** [comment]
Thanks Timo! Could you change the html address that appears at the RHS of the page on https://github.com/geodynamics/burnman/?

**Comment by tjhei** [comment]
I made burnman.org redirect there automatically. Do you still want me to change it?

**Pull request merged**